File: AppGiaoHangAPI.Model/Model/CustomerOrderInformation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace AppGiaoHangAPI.Model.Model
{
    public partial class CustomerOrderInformation
    {
        public CustomerOrderInformation()
        {
            CustomerOrders = new HashSet<CustomerOrder>();
        }
        [Key]

        public long CustomerOrderInformationId { get; set; }
        public long? CustomerId { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Address { get; set; }

        public virtual Customer? Customer { get; set; }
        public virtual ICollection<CustomerOrder> CustomerOrders { get; set; }
    }
}

File: AppGiaoHangAndroid/app/src/main/java/com/pnb309/appgiaohang_android/Entity/CustomerOrderInformation.java
package com.pnb309.appgiaohang_android.Entity;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class CustomerOrderInformation implements Serializable {
    public CustomerOrderInformation() {
        CustomerOrders = new ArrayList<>();
    }

    private long CustomerOrderInformationId;
    private Long CustomerId;
    private String PhoneNumber;
    private String Address;

    private Customer Customer;
    private List<CustomerOrder> CustomerOrders;

    public long getCustomerOrderInformationId() {
        return CustomerOrderInformationId;
    }

    public void setCustomerOrderInformationId(long customerOrderInformationId) {
        CustomerOrderInformationId = customerOrderInformationId;
    }

    public Long getCustomerId() {
        return CustomerId;
    }

    public void setCustomerId(Long customerId) {
        CustomerId = customerId;
    }

    public String getPhoneNumber() {
        return PhoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        PhoneNumber = phoneNumber;
    }

    public String getAddress() {
        return Address;
    }

    public void setAddress(String address) {
        Address = address;
    }

    public com.pnb309.appgiaohang_android.Entity.Customer getCustomer() {
        return Customer;
    }

    public void setCustomer(com.pnb309.appgiaohang_android.Entity.Customer customer) {
        Customer = customer;
    }

    public List<CustomerOrder> getCustomerOrders() {
        return CustomerOrders;
    }

    public void setCustomerOrders(List<CustomerOrder> customerOrders) {
        CustomerOrders = customerOrders;
    }
}
